app/Http/Controllers/CelebController.php

```diff
@@ -22,14 +23,16 @@ public function create()
         return view('celebs/create');
     }
 
-    public function store(Request $request)
+    public function store(CelebRequest $request)
     {
-        //
+        Celeb::create($request->validated());
+
+        return redirect('celebs');
```

Comment:
You could probably have a success message appear after you redirect to the celebs. index.